chanreg.md

```diff
@@ -1,228 +1,196 @@
-representation on our services and relationship with us.
+Our [policies](/policies) outline that channels on Libera.Chat fall into one
+of three categories and two namespaces. Channels that begin with only a single
+`#` character are for on-topic project and community channels (e.g. `#linux`).
+Channels that begin with two or more `#` characters are for informal channels
+(e.g. `##chat`).
 
-### On-topic projects
+### What is on-topic?
 
 Projects considered to be on-topic for Libera.Chat are primarily free and
 open-source software projects, and other peer-directed projects, for instance
 Linux User Groups (LUGs), student societies, hacker- or makerspaces and other
-collaborative efforts. It can also cover projects/companies of general
-interest to our user base.
-
-### Claim to the name and representation
-
-A claim to the name is given if your project is on-topic as per the above and
-named like that, or commonly called / abbreviated that. In case of name
-clashes, e.g. other on-topic projects already present using the same name or
-there being obvious trademark issues, our team will try to find a good
-solution. The same applies if your name contains characters that can't be used
-on our services for technical reasons.
-
-The registration should be either done by or have received the official
-blessing of someone who is able to speak for the project. This could be:
+collaborative efforts. At staff discretion, it can also cover projects or
+companies of general interest to our user base.
+
+If your channel fits the above criteria and you are or have the official
+blessing of someone who is able to speak for the project or group, then you
+would qualify for an on-topic project channel. If your channel fits the above
+criteria but is otherwise unofficial, then you would qualify for an on-topic
+community channel. If your channel does not fit the above criteria, and it is
+not otherwise
+[off-topic for Libera.Chat](/policies/#unwanted-content-and-behaviour), then
+you would qualify for an informal channel.
+
+### Expiration and disputes
+
+On-topic project channels and on-topic community channels do not expire. An
+official claim to the name of an on-topic community channel is not needed, but
+not having it bears the risk of losing the channel should an on-topic entity
+with a valid claim start an official presence on Libera.Chat. Staff will try
+to find a good solution for both parties in such cases.
+
+Informal channels are given out on a first-come, first-served basis. They will
+expire if unused for a long time, see our [policies](/policies/#channels) for
+details regarding when this occurs. You can
+[contact a staff member on IRC](/guides/faq#how-to-find-libera-chat-staff)
+with a request to take over an expired informal channel; they are not
+automatically deleted once they expire.
+
+## Formal group registration
+
+Established on-topic projects and communities are able to formally register
```

Comment:
Is this a policy change? I think it's probably worth asking projects to be _established_ but I don't know that it's been a very hard requirement in the past. I'm :+1: on the new language.